sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -51,18 +61,25 @@ def __init__(self, dataflow_client=None, job_result=None, job_graph=None):
     self._cached_metrics = None
     self._job_graph = job_graph
 
+  @staticmethod
+  def is_counter(metric_result):
+    return isinstance(metric_result.attempted, numbers.Number)
```

Comment:
I followed advice here: http://stackoverflow.com/questions/4187185/how-can-i-check-if-my-python-object-is-a-number?answertab=active#tab-top
It seems that this way is more general. What do you think?